EventStore/EventStore-Client-NodeJS
# Persistent Subscriptions to $all

## Client
- Add server features support
- Align persistentSubscriptionSettings [breaking change] [RFC022](https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/022%20-%20gRPC%20Persistent%20Subscriptions.md#defaults-and-parameter-descriptions)
- `createPersistentSubscriptionToAll`
- `updatePersistentSubscriptionToAll`
- `deletePersistentSubscriptionToAll`
- `connectToPersistentSubscriptionToAll`
- make `TwoWaySubscription` generic, and rename to` PersistentSubscriptionImpl`

## tests
- Add version check environment for skipping tests based on version
- Update tests to use new `persistentSubscriptionSettings` shape
- Tests for added methods
- Make sure a leader has been elected before starting tests
